lib/PuppeteerSharp/Helpers/JsonHelper.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Serialization;
+
+namespace PuppeteerSharp.Helpers
+{
+    internal static class JsonHelper
+    {
+        public static JsonSerializerSettings DefaultJsonSerializerSettings = new JsonSerializerSettings
+        {
+            ContractResolver = new CamelCasePropertyNamesContractResolver()
```

Comment:
shouldn't this include `NullValueHandling = NullValueHandling.Ignore`?

Reply:
Did that, next commit :)